### ISSUE TITLE
Alternative to YouTubeCenter

### ISSUE THREAD

**Issue by p1nkyy** [YePpHa/YouTubeCenter#2417]
If you're looking JUST to prevent video autoplay and disable annotations then you can use Youtube+ (https://github.com/ParticleCore/Particle).
To disable DASH playback (or enable auto buffer without playing the video) you can try this trick I use: https://beebom.com/how-force-youtube-fully-buffer-video-chrome-firefox/ BUT you can ONLY watch video up to 720p if you do this.

DISADVANTAGES:
- Youtube+ didn't automatically show the start of the video like YTC did, had to press the red play button manually and then pause it to start the buffering process.
- The video in theatre mode has black background. I didn't like it, it used to be white when I used YTC and I can't figure out how to make it white again.

Every other features, basically Youtube+ has it. AFAIK the only thing it doesn't have is the 'Disable DASH playback'.

**Comment by Magnic** [comment]
I changed today to enhancer (yes, it's not the full name, i won't post it's full name) and i have no issues anymore. Of course it doesn't have so many nice options as YTCenter, but it does work and doesn't break.

**Comment by p1nkyy** [comment]
@Magnic what script are you talking about? Is it the same with the one I discussed? What is the reason of not sharing?

**Comment by Magnic** [comment]
I don't use a script, i use an addon. A Firefox addon. With this info you should be able to find it in 10s.

**Comment by NiGHTsC** [comment]
Sounds like **YouTube Enhancer Plus** or for FF57+ **Enhancer for YouTube**

**Comment by Symbai** [comment]
As far as I can seee Enhancer for Youtube is closed source. Since so many popular addons turned out into selling your data, I don't see why this is any alternative at all.

**Comment by andrewpros** [comment]
Now after extensive testing i highly recommend Youtube+ instead of Enhancer for YouTube, the later has some nasty slowdowns in fx even if you are not on YT, or is it only me, but as soon as i changed to YT+ it is all gone.

And also YT+ properly stops the auto play while Enhancer plays some of it and then stops after 1-2 sec, very annoying and i can only guess that YT+ has a better way of doing things.

**Comment by Symbai** [comment]
Youtube+ Chrome extension has been sold and infected by malware. Since the author of it hasn't informed anyone BEFORE selling it, I wouldn't really recommend using anything from this guy anymore as you can never be sure if he wouldn't do this again for money, but its up to you.

**Comment by andrewpros** [comment]
@Symbai Anything to back that up?

**Comment by Symbai** [comment]
https://github.com/ParticleCore/Particle/issues/504
https://www.bleepingcomputer.com/news/security/-particle-chrome-extension-sold-to-new-dev-who-immediately-turns-it-into-adware/

**Comment by andrewpros** [comment]
@Symbai 

Thank you very much, this clears some things.

I will still use the extension, it is safe, i'm using the FX version, as explained by the author here https://github.com/ParticleCore/Particle/issues/528

Only chrome extension was sold and everyone still can use the original as a tampermonkey script, on fx i'm perfectly safe, don't have auto update on it either.

And it is also important to note that current YT+ status is EOL as new YT layout will make it unusable.

There is already a new addon in works https://github.com/ParticleCore/Iridium

**Comment by Symbai** [comment]
I know. But if someone sold an open source addon with a large user base and do not inform the users about it (a contract that forbids that should be questionable anyways) should be responsible as well about what happens with the addon afterwards.

If he goes for money because no one donated. Then I would ask myself what he does if his new addon doesn't get the desired donations as well. Will he sell it again?

Therefore, I would be careful with a recommendation, especially without to mention this. But its up to everyone himself what alternative he wants to use.

**Comment by andrewpros** [comment]
No, he is not responsible for what the new owner have done. He has learned his lesson, if he sells it again, this time he will inform about it, there is an obvious reason new owners don't want to know they bought it.
He could be smarter, yes, but the source is there, he confessed everything, no on is forcing you to auto update an addon, i update only manually, so there is ZERO possibility i will get any new version with changes without my review.
People always like to blame everybody not seeing their own faults and lack of being smart, they just install everything next, next, next, without any clue, i couldn't care less about such people, they just mindless, they don't want to be cautious, that's fine, but if they get some malware, it is also their fault.

**Comment by ParticleCore** [comment]
@Symbai 

>Make no mistake, **if I am offered another deal I will take it again, but this time I will post the emails publicly for the duration of the exchange (with sensitive information redacted, such as names and emails) so that this won't catch users by surprise again.**

https://github.com/ParticleCore/Particle/issues/528#issuecomment-315840535

Also

>2 - The income is very important during the period that my life is currently on. I do appreciate all the donations that have been made and keep being made since the birth of this project, but with just **around $8 average per month** (if I recall correctly) there's just not enough, not even to pay the ISP.

https://github.com/ParticleCore/Particle/issues/528#issue-242183983 (literally the first post)

Instead of making assumptions, like fear mongering or stating that no one donated, take some actual time to read what I already explained, some things even more than once.

Also do not use this issue tracker for this kind of discussion, this is about alternatives to YTC (which I am not really ok with, but it is not my issue tracker), please try to stay in the subject.

**Comment by TheEssemCraft** [comment]
What about viewing the resolution/framerate/like bar on the thumbnail?

**Comment by rikkdekkard** [comment]
@TheEssemCraft 
> What about viewing the resolution/framerate/like bar on the thumbnail?

I'd recommend this:
[https://greasyfork.org/en/scripts/31663-youtube-video-preview-player](url)

**Comment by andrewpros** [comment]
I just watch videos that interests me, i don need resolution/framerate/like bar on the thumbnail.
